cmds/capability/capability.go

```diff
@@ -50,3 +66,24 @@ var capabilityByIdCmd = &cobra.Command{
 
 	},
 }
+
+var createCapabilityCmd = &cobra.Command{
+	Use:   "create [NAME]",
+	Short: "creates a new capability",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			fmt.Println("Missing id")
+			os.Exit(1)
+		}
+		capabilityStruct := selfservice.CapabilityRequest{
+			Name:         args[0],
+			Description:  description,
+			Invitees:     invitees,
+			JsonMetadata: metadata,
+		}
+		fmt.Println(capabilityStruct)
```

Comment:
Do we want this and the following?
They look like debugging -- and if so, maybe we should check for the 'verbose' flag?

Reply:
The first one is reasonable, but without the second one we don't have an output

Reply:
We can also simply keep both. Up to you.
We will change how we do output at a later stage anyway :)